### DISCUSSION TITLE
Delays on evaluations

### DISCUSSION THREAD

**Discussion by oDasBeiras** [OpenMined/courses#265] [Course: Our Privacy Opportunity] [answered]
Hello. Submitted my assignment 13 days ago. Is this delay expected? Thank you.

Best regards
André

**Comment by yemikifouly** [comment, marked as answer]
Hello André! Expected no. Not at all. Could you please share your last name? I will check our side what the issue might be. Thank you. 🙂

**Comment by oDasBeiras** [comment]
Hello.

Thank you for the swift reply.

André Castro

**Reply by yemikifouly** [reply]
Hi again André! Your project was indeed successfully submitted. The issue was on our side and is now being dealt with. Our mentors will review your project and you should expect feedback very soon. 🙂 Thank you for bringing this to our attention. I hope you enjoyed the course very much.

**Comment by oDasBeiras** [comment]
Oh, ok. No problem. The course is incredible. Opened up a lot of new
perspectives for me. Thank you very much.


Best regards
André Castro

**Comment by oDasBeiras** [comment]
Hello again.

I've just received feedback, but when trying to click complete the course,
got an error message (sorry, did not get a print screen of it).

[image: image.png]


Upon refreshing, I can no longer access my feedback and my progress is
wrong:

[image: image.png]

Can you help me? Thank you.


Best regards
André Castro

**Reply by yemikifouly** [reply]
Hello André! 👋 I am really glad you received feedback and passed the project! Congratulations!!! 👏👏👏 

About the issue you are now encountering, it is due to some changes we made to the course structure last week. It is being fixed. I mention it in **[this post](https://github.com/OpenMined/courses/discussions/263#discussion-3263490)**. Based on the updates from our Web & Mobile Team, the issue should be fixed this week, probably in the next couple of days. 🙂 

Thank you for bearing with us while it is being fixed. 🙂

**Reply by yemikifouly** [reply]
Happy Tuesday André! 👋  The issue with the progress in the classroom has been fixed. Just stopping by to see if everything is working well now. Has your progress been restored? Can you access your feedback?

**Comment by oDasBeiras** [comment]
Hello Yemissi.

First of all, your student support is better than any customer support of
paid products I've ever encountered. Congratulations and thank you!

The progress is now correctly displayed. I believe I'm supposed to click
finish

[image: image.png]

When I do, the following happens

[image: image.png]

Upon clicking Continue

[image: image.png]


Best regards
André Castro

**Reply by tcp** [reply]
Hi @oDasBeiras!

We have deployed this fix today. Please try again — should work.

Thanks!

**Comment by oDasBeiras** [comment]
Hello.

The issue still persists, but it's different now.

[image: image.png]
After clicking continue:

[image: image.png]

It redirects me to the middle of the course.


Thank you.


Best regards
André Castro

**Reply by tcp** [reply]
@oDasBeiras André, none of your images were uploaded :(

This is the only report we've had of problems such as this after rolling out some adjustments last week. I'd like to investigate further. Can you join OM's Slack or [log your issue here](https://github.com/OpenMined/openmined/issues) with more details?
